crates/biome_js_analyze/src/analyzers/nursery/no_default_export.rs

```diff
@@ -0,0 +1,82 @@
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic};
+use biome_console::markup;
+use biome_js_syntax::AnyJsExportClause;
+use biome_rowan::AstNode;
+
+declare_rule! {
+    /// Disallow `export default`.
```

Comment:
The rule is about "default exports" in general, NOT about `export default`. If we document it like this, users might think that grammar like `export { module as default }` is legal but it shouldn't